.github/workflows/ci-frontend.yml

```diff
@@ -0,0 +1,64 @@
+name: Frontend Test & Lint
+
+on:
+  push:
+    paths:
+      - 'frontend/**'
```

Comment:
The docker-compose file itself may also affect the unit tests, in case commands or dependencies change.